README.md

```diff
@@ -85,15 +85,30 @@ http://192.168.99.100:30080/api/prom/api/v1/query?query=up
 
 If that doesn't work, your Minikube might be using a different ip address. Check with `minikube status`.
 
-### Vendoring
+### Dependency management
 
-We use `dep` to vendor dependencies. To fetch a new dependency, run:
+We uses [Go modules](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) to manage dependencies on external packages. This requires a working Go e
+nvironment with version 1.11 or greater, git and [bzr](http://wiki.bazaar.canonical.com/Download) installed.
```

Comment:
Please don't break lines in the middle of words.

Reply:
Fix it.